includes/class-create-block-theme-editor-tools.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * The Editor integration for the Create Block Theme plugin.
+ * @since      2.2.0
+ * @package    Create_Block_Theme
+ * @subpackage Create_Block_Theme/includes
+ * @author     WordPress.org
+ */
+class Create_Block_Theme_Editor_Tools {
```

Comment:
Let's keep the CBT class name prefix we've established.

Reply:
I actually wrote this part before we added the namespacing.  Thanks for pointing that out. :)